src/utils/number.ts

```diff
@@ -23,7 +23,7 @@ export const localeNumberString = (value: BigNumber | string | number): string =
   return origin.isNegative() ? `-${text}` : text
 }
 
-const MIN_VALUE = new BigNumber(10 ** 3)
+const MIN_VALUE = new BigNumber(1)
```

Comment:
will the numbers on the chart change together?

Reply:
All difficulties and hash rates format will be changed

Reply:
cool